specification/datalake-analytics/resource-manager/readme.md

```diff
@@ -39,8 +39,6 @@ These settings apply only when `--tag=package-2016-11` is specified on the comma
 ``` yaml $(tag) == 'package-2016-11'
 input-file:
 - Microsoft.DataLakeAnalytics/2016-11-01/account.json
-- Microsoft.DataLakeAnalytics/2016-11-01/catalog.json

```

Comment:
I believe these files are data plane and that's why we removed them in ruby generation of mgmt gem, I'm not sure it's the case for other languages, you may want to confirm with .net if that's the case for them too.